packages/wmr/src/lib/prerender.js

```diff
@@ -172,6 +172,12 @@ async function workerCode({ cwd, out, publicPath }) {
 			if (result.head) {
 				head = result.head;
 			}
+
+			if (result.data && typeof result.data === 'object') {
+				body += `<script type="isodata">${JSON.stringify(result.data)}</script>`;
```

Comment:
Let's add a simple test similar to the `prerender-external` fixture that just checks if the passed data string shows up in the rendered html 👍